command/alloc_status.go

```diff
@@ -498,7 +496,7 @@ func buildDisplayMessage(event *api.TaskEvent) string {
 		desc = strings.Join(parts, ", ")
 	case api.TaskRestarting:
 		in := fmt.Sprintf("Task restarting in %v", time.Duration(event.StartDelay))
-		if event.RestartReason != "" && event.RestartReason != restarts.ReasonWithinPolicy {
+		if event.RestartReason != "" && event.RestartReason != "Restart within policy" {
```

Comment:
Can we copy this into `api` too, so that we don't have a raw string floating around?